examples/metrics/prometheus.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This module serves as an example for a simple application using metrics
+Examples show how to recording affects the collection of metrics to be exported
+"""
+from opentelemetry import metrics
+from opentelemetry.ext.prometheus import PrometheusMetricsExporter
+from opentelemetry.sdk.metrics import Counter, Meter
+from opentelemetry.sdk.metrics.export.controller import PushController
+
+# Meter is responsible for creating and recording metrics
+meter = Meter()
+metrics.set_preferred_meter_implementation(meter)
```

Comment:
I believe `set_preferred_meter_implementation` takes in a `meter_factory` rather than an actual `meter`.

https://github.com/open-telemetry/opentelemetry-python/blob/master/examples/metrics/record.py#L27